README.md

```diff
@@ -7,6 +7,9 @@
   <a href="https://github.com/tobealive/wthrr-the-weathercrab/actions/workflows/ci.yml?query=branch%3Amain" target="_blank">
     <img alt="crates.io" src="https://img.shields.io/github/actions/workflow/status/tobealive/wthrr-the-weathercrab/ci.yml?branch=main&style=flat-square" />
   </a>
+  <a href="https://github.com/tobealive/wthrr-the-weathercrab" target="_blank">
+    <img alt="github.com" src="https://img.shields.io/github/last-commit/tobealive/wthrr-the-weathercrab?style=flat-square" />
```

Comment:
```suggestion
    <img alt="last-commit" src="https://img.shields.io/github/last-commit/tobealive/wthrr-the-weathercrab?style=flat-square" />
```

The workflow status alt is a residue from copy pasting I see. It could be e.g. `"workflow-status"` or `"build-status"`. I think this fits here so the commit updates the badges generally 🙂

Reply:
:wrench: :heavy_check_mark: